mod-source-record-manager-server/src/main/java/org/folio/services/ChangeEngineServiceImpl.java

```diff
@@ -253,8 +255,22 @@ private boolean createInstanceActionExists(JobExecution jobExecution) {
       Action.CREATE);
   }
 
+  private boolean createAuthorityActionExists(JobExecution jobExecution) {
+    return containsMarcActionProfile(
+      jobExecution.getJobProfileSnapshotWrapper(),
+      List.of(FolioRecord.AUTHORITY),
+      Action.CREATE);
+  }
+
+  private boolean createMarcHoldingsActionExists(JobExecution jobExecution) {
```

Comment:
this method returns boolean, I think there should be better to name this method smth like "isCreateMarcHoldingsActionExists". What do you think?

Reply:
@Aliaksandr-Fedasiuk  Done.